db/config.cc

```diff
@@ -468,6 +468,8 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     /* Cache and index settings */
     , column_index_size_in_kb(this, "column_index_size_in_kb", value_status::Used, 64,
         "Granularity of the index of rows within a partition. For huge rows, decrease this setting to improve seek time. If you use key cache, be careful not to make this setting too large because key cache will be overwhelmed. If you're unsure of the size of the rows, it's best to use the default setting.")
+    , column_index_auto_scale_threshold_in_kb(this, "column_index_auto_scale_threshold_in_kb", value_status::Used, 10240,
```

Comment:
Can you make this LiveUpdate? To avoid rolling restarts.

Reply:
Never done this, but I guess we can. Will do.